SimpleMVVM/API Service/Networking.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Networking.swift
+//  SimpleMVVM
+//
+//  Created by Дмитрий Станкевич on 10.02.22.
+//
+
+
+import Foundation
+import UIKit
```

Comment:
`import Foundation` is redundant. `import UIKit` already imports `Foundation` inside itself

Reply:
right click on UIKit + `show definition ` and u will see it